app/views/users/edit.html.erb

```diff
@@ -95,6 +95,10 @@
           hint: "Enables a more compact and less guided post uploader." %>
         <% end %>
 
+        <%= f.input :use_new_search_syntax, :as => :select, label: "Use New Search Syntax",
+                    hint: raw("Enables the use of the new search syntax."),
```

Comment:
you don't need this raw() here

Reply:
Fixed c5d336ccffd0730223b2a6576963a76bde89f614